universal-application-tool-0.0.1/app/services/applicant/question/FileUploadQuestion.java

```diff
@@ -17,11 +17,16 @@
 public class FileUploadQuestion implements Question {
 
   private final ApplicantQuestion applicantQuestion;
-  private Optional<String> fileKeyValue;
-  private Optional<String> originalFileNameValue;
+  private Optional<Optional<String>> fileKeyValue;
+  private Optional<Optional<String>> originalFileNameValue;
 
   public FileUploadQuestion(ApplicantQuestion applicantQuestion) {
     this.applicantQuestion = applicantQuestion;
+    // This value is serving double duty as a singleton load of the value.
```

Comment:
Suggestion:

I'd put this on the class members bove since they have the semantics instead of this one location that sets them.

Also, in looking at other code I was cleaning them up by also adding Cache to the end of the name to help convey their multi purpose use.

Reply:
In order to get this working, looks like I'll have to change at least 6 test files. Should I make these changes in a separate PR or is that fine with you? I'm just a little worried about this PR becoming too big but happy to make the fixes if you'd like

Reply:
What you have here is what I expected.

If you thought I meant for you to add it in the other unrelated uses, I didn't.  I was conveying that I had been adding Cache to those names to add clarity.